docker-for-mac/release-notes.md

```diff
@@ -16,10 +16,29 @@ This page contains information about the new features, improvements, known issue
 > Starting with Docker Desktop 3.0.0, Stable and Edge releases are combined into a single release stream for all users. Updates to Docker Desktop will now be available automatically as delta updates from the previous version. This means, when there is a newer version of Docker Desktop, it will be automatically downloaded to your machine. All you need to do is to click **Update and restart** from the Docker menu to install the latest update.
 {: .important }
 
+
+## Docker Desktop Community 3.1.0
+2021-01-14
+
+> [Download](https://desktop.docker.com/mac/stable/Docker.dmg)
+
+### Upgrades
+
+- Docker daemon now runs within a Debian Buster based container
+- [Compose-CLI 1.0.7](https://github.com/docker/compose-cli/tree/v1.0.7)
+
+### Bug fixes and minor changes
+
+  - Fixed a lot of UI reliability issues
```

Comment:
I think if we're going to put this, we need to say more about it. What were the reliability issues? Are they visible to the user? Are there any tickets in public repositories?

Reply:
Well it is mostly everything related to events coming from the engine in burst. What about `Fixed UI reliability issues when a lot of events are generated by the daemon` ?

Reply:
Were there user-reported bugs that we think are due to this, or was it only in CI?

Reply:
I don't think with have user-reported bugs for that (mostly because those are transient bugs - just closing / re-opening the UI fixes it)